generator/geo_objects/geo_objects.cpp

```diff
@@ -1,4 +1,5 @@
 #include "generator/geo_objects/geo_objects.hpp"
+#include "generator/geo_objects/key_value_storage.hpp"
```

Comment:
Empty line after the first line, please.

Reply:
done